<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>热化学反应解析器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="toolTip">
       <string notr="true"/>
      </property>
      <property name="placeholderText">
       <string>输入方程式 详见关于(H)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit2">
      <property name="placeholderText">
       <string>输入条件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>输出结果</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>解析</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>关于(&amp;H)</string>
    </property>
    <addaction name="action_H"/>
    <addaction name="separator"/>
    <addaction name="action_D"/>
    <addaction name="action_Qt"/>
    <addaction name="action_this"/>
   </widget>
   <widget class="QMenu" name="menu_R">
    <property name="title">
     <string>反馈(&amp;R)</string>
    </property>
    <addaction name="action_github"/>
    <addaction name="action_bug"/>
   </widget>
   <addaction name="menu_H"/>
   <addaction name="menu_R"/>
  </widget>
  <action name="action_github">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>源代码获取</string>
   </property>
  </action>
  <action name="action_bug">
   <property name="text">
    <string>报告bug</string>
   </property>
  </action>
  <action name="action_H">
   <property name="text">
    <string>使用辅助</string>
   </property>
  </action>
  <action name="action_D">
   <property name="text">
    <string>关于开发</string>
   </property>
  </action>
  <action name="action_Qt">
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="action_this">
   <property name="text">
    <string>关于解析器</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
